src/shippingState.js

```diff
@@ -123,7 +141,28 @@ const addBagToBox = (boxId, boxToUpdate) => {
         allBoxesList.push(boxToUpdate);
     }
 
-    appState.setState({ allBoxesList: allBoxesList });
+    const strayTubesToAdd = getStrayTubesFromUncheckedModalBoxes(boxToUpdate, bagId);
+
+    if (strayTubesToAdd.length > 0) {
+        if (!availableCollectionsObj['unlabelled']) {
+            availableCollectionsObj['unlabelled'] = [];
+        }
+        availableCollectionsObj['unlabelled'].push(...strayTubesToAdd);
+    }
+
+    appState.setState({
```

Comment:
{A:A, B:B} can be shortened to {A, B}

Reply:
Nice -> ES6 and later, when (property name === variable name), we can use the shorthand property name syntax. I wasn't aware of this. Thanks.

Reply:
Updated